package.json

```diff
@@ -107,7 +107,7 @@
 		"semver": "^5.3.0"
 	},
 	"peerDependencies": {
-		"jest": "^18.0.0 || ^19.0.0 || ^20.0.1"
+		"jest": "^18.0.0 || ^19.0.0 || ^20.0.1 || ^20.1.0-a"
```

Comment:
why 20.1 alpha, as opposed to `^20.1.0-echo` or `^21.0.0-beta`?

Reply:
AFAIK there isn't documentation yet on what the breaking changes are in 21, but I did verify that the fix we're looking for is in the 20.1 prerelease versions. I picked a semver string that would match all of the 20.1 prereleases, the -a will match against -alpha.1, -alpha.2, -echo.x, etc.

We can try to also include 21, but we'd need to verify we're unaffected by breaking changes.

Reply:
i'm first asking why even allow 20.0.1 a-d prereleases, as opposed to requiring echo or higher - and then separately, if we can make jest-wrap work with v21, then we can skip supporting v20 prereleases entirely